### DISCUSSION TITLE
I don't understand how to run the examples

### DISCUSSION THREAD

**Discussion by alexvedi** [needle-tools/needle-engine-samples#1] [Q&A] [answered]
Hi

Sorry, but I don't understand how to run the examples. Can you give some more information? maybe a simple step by step?. 

:(

**Comment by marwie** [comment, marked as answer]
Hello,

- you download the repository using (ideally pull it using git)
- then open the Unity project in the "projects/" folder
- a window will open showing you the samples :)

hope it helps
